### ISSUE TITLE
Allow "dummy" ticket ids in commiticketing

### ISSUE THREAD

**Issue by sleicht** [ShellMagick/commit-hooks#26]
I know that it goes against the original intent. But sometimes you simply don't have a ticket or are in a repository where the current ticket number doesn't make sense. For such cases, I sometimes use ticket numbers like TICKET-0000. This should also be allowed.
Sure, in the command line, the check can be temporarily disabled, but sometimes you're in a GUI or don't know the option right now.

**Comment by AFulgens** [comment]
Indeed, it kind of goes against the original intent of the hook. However, in case you are serious about the original intent, you would use `cojira` anyways, so I'm not 100% against relaxing the regex to allow `[0-9]*` instead of `[1-9][0-9]*`.
